skimage/filters/_gaussian.py

```diff
@@ -155,3 +155,117 @@ def _guess_spatial_dimensions(image):
         return 3
     else:
         raise ValueError("Expected 2D, 3D, or 4D array, got %iD." % image.ndim)
+
+
+def difference_of_gaussians(image, sigma1, sigma2=None, *,
+                            mode='nearest', cval=0, multichannel=False,
+                            truncate=4.0):
+    """Multi-dimensional band-pass filter using the Difference of Gaussians
+    method.
```

Comment:
This should fit in one line, with additional comments separated by a blank line. Also, "multidimensional band-pass filter" is very jargony. I would suggest something like:

```
Find features between `low_sigma` and `high_sigma` in size.
```

You can expand on band-pass filters all you like in a subsequent paragraph. =)